Exercises/Shreeya, Tulabandula/task_04_shreeya.py

```
@@ -0,0 +1,146 @@
+
+def param_modes(memory, position, index_nr, mode):
+    if mode == 1:
+        return memory[position + index_nr]
+    elif mode == 0:
+        return memory[memory[position + index_nr]]
+    else:
+        print("Error!")
```

Maybe raise an actual error which gives the user a bit more information about the type of error encountered.